moby-tini/mapping.go

```diff
@@ -69,7 +66,7 @@ var (
 			"/build/src/build/tini-static",
 		},
 		Description: BaseArchive.Description,
-		Conflicts:   []string{},
+		Conflicts:   []string{"moby-engine < 24.0.0"},
```

Comment:
Not needed.

Reply:
Because of the destination change for the `docker-init` executable?

Reply:
Yep.

Reply:
done